src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,12 @@
-export const CommentList = () => <>Put the list here</>;
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length !== 0
+      ? comments.map(comment => (
+          <CommentInfo comment={comment} key={comment.id} />
+        ))
+      : null}
```

Comment:
```suggestion
    {!!comments.length
      && comments.map(comment => (
          <CommentInfo comment={comment} key={comment.id} />
        ))}
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

Reply:
Done!! Сorrected!)